scripts/fast-build/config-update.bat

```diff
@@ -7,4 +7,4 @@ xcopy /e /h /i /q /y %dspace_source%\dspace\config\ %dspace_application%\config\
 
 cd %dspace_source%\scripts\fast-build\
 
-call update-solr-configsets.bat
+REM call update-solr-configsets.bat
```

Comment:
why ?

Reply:
Because you need to restart Solr after updating it's config-sets. And I do not want to do it every time after copying configs. And you rarely must to update config-sets. In the past I have one issue where I needed to refresh config-sets and I thought that it could be great idea to add it into my custom `fast-config-update` script. And now I see it wasn't so great.